README.md

```diff
@@ -77,7 +81,62 @@ variables to your root queries and containers:
   component={Widgets}
   queries={WidgetsQueries}
   queryParams={['date', 'color']} // date and color will be passed as variables
-/>`
+/>
+```
+
+If you’re passing on params from path components, like a URL containing slugs, make sure to prefix your params with a colon:
```

Comment:
You can see how `queryParams` get used here: https://github.com/relay-tools/react-router-relay/blob/v0.4.2/src/getParamsForRoute.js#L14-L25. It only interacts with `location.query`.